pkg/osquery/tables/dev_table_tooling/commands_windows.go

```diff
@@ -3,9 +3,15 @@
 
 package dev_table_tooling
 
+import "path/filepath"
+
 var allowedCommands = map[string]allowedCommand{
 	"echo": {
 		binPaths: []string{"echo"},
 		args:     []string{"hello"},
 	},
+	"cb_repcli": {
+		binPaths: []string{filepath.Join("Program Files", "Confer", "repcli")},
```

Comment:
This doesn't need `C:` prepended?

Reply:
I don't think so? Go seems pretty chill about that